meetings/2025-03-04-taskforce-meeting.md

```diff
@@ -0,0 +1,26 @@
+# 2025-03-04: ActivityPub Trust and Safety Task Force Meeting
+
+## Attending
+
+- Emelia Smith (thisismissem@hachyderm.io)
+- Dmitri Z.
+- Jaz-Michael King (https://mastodon.iftas.org/@jaz)
+- Evan Prodromou (acct:evan@cosocial.ca)
```

Comment:
@ChaosExAnima you might want to add yourself in here by using a change suggestion, and then I can commit that.

Reply:
```suggestion
- Evan Prodromou (acct:evan@cosocial.ca)
- Echo (https://tech.lgbt/@chaosexanima)
```